benchmarks/conftest.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Pytest configuration file for PennyLane-Benchmarks suite.
+"""
+import os
+
+import pennylane as qml
+
+# Looking for the device for benchmarking.
+default_device = "lightning.qubit"
+supported_devices = {"default.qubit", "lightning.qubit"}
+supported_devices.update({sb.replace(".", "_") for sb in supported_devices})
```

Comment:
Why do you need this device string update?

```suggestion
supported_devices = {"default.qubit", "lightning.qubit"}
```

Reply:
So one can call with both 'lightning_qubit' and 'lightning.qubit', this is particularly useful when running workflows that also build Lightning.